### PR TITLE
[Notifier] Fix ContactEveryoneOptions

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Thanks to @franckranaivo in https://github.com/symfony/symfony/pull/50349#discussion_r1210200114

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php [hunk 1]
```diff
@@ -35,7 +37,7 @@ public function getRecipientId(): ?string
      */
     public function diffusionName(string $diffusionName): static
     {
-        $this->options['diffusion_name'] = $diffusionName;
+        $this->options['diffusionname'] = $diffusionName;
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:40]: No fix in README needed?

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php [hunk 2]
```diff
@@ -35,7 +37,7 @@ public function getRecipientId(): ?string
      */
     public function diffusionName(string $diffusionName): static
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:38]:
```suggestion
    public function diffusionname(string $diffusionName): static
```
as it is not released yet?

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php [hunk 3]
```diff
@@ -50,6 +52,28 @@ public function category(string $category): static
         return $this;
     }
 
+    /**
+     * @param 'fr_FR'|'en_GB' $locale
+     *
+     * @return $this
+     */
+    public function locale(string $locale): static
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:60]: should be added to the README

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php [hunk 4]
```diff
@@ -50,6 +52,28 @@ public function category(string $category): static
...
+    {
+        $this->options['locale'] = $locale;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function unicode(bool $unicode): static
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:70]: should be added to the README

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php (hunk 2 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:38]: Wouldn't fit our CS, we use camelcase

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php (hunk 3 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:60]: no need to IMHO, the code says it all

File: src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php (hunk 4 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Notifier/Bridge/ContactEveryone/ContactEveryoneOptions.php:70]: neither :)

**Review by OskarStark — APPROVED** [review]

**Review by mtarld — APPROVED** [review]

**Pull request merged**